Rubberduck.Core/UI/Settings/SettingsForm.cs

```diff
@@ -45,42 +46,43 @@ public SettingsForm()
                      *     .SelectMany(s => s.DefinedTypes)
                      *     .Where(w => Attribute.IsDefined(w, typeof(ExperimentalAttribute)))
                      */
-                    Control = new GeneralSettings(new GeneralSettingsViewModel(config, operatingSystem, messageBox, vbeSettings, new List<Type>())),
+                    //Control = new GeneralSettings(new GeneralSettingsViewModel(config, operatingSystem, messageBox, vbeSettings, new List<Type>())),
+                    Control = new GeneralSettings(viewModelFactory.Create<Rubberduck.Settings.GeneralSettings>(config)),
                     View = SettingsViews.GeneralSettings
                 },
                 new SettingsView
                 {
-                    Control = new TodoSettings(new TodoSettingsViewModel(config)),
+                    Control = new TodoSettings(viewModelFactory.Create<ToDoListSettings>(config)),
```

Comment:
Yes it could be made into its own factory, similar to what I did with the viewmodels. However, that would broaden the scope of the PR. I want to do that in its own PR, too and preferably after #4072 is merged, since the changes should be more universal than just the refactoring dialogs (e.g. abstraction of the WinForm out of the picture, having a more uniform system of activating a dialog)